tools/generate_json_rpc_docs.py

```diff
@@ -133,10 +130,8 @@ def to_markdown(self):
         """
         @return: str containing the Markdown table
         """
-        output = []
-        output.append("| Name | Type | Description |")
-        output.append("| --- | --- | --- |")
-        tag_re = re.compile(r"^\{(\w+)\}\s+([\S]+)\s+-\s+(.*)$", re.DOTALL)
+        output = ["| Name | Type | Description |", "| --- | --- | --- |"]
+        tag_re = re.compile(r"^{(\w+)}\s+(\S+)\s+-\s+(.*)$", re.DOTALL)
```

Comment:
[\S] vs \s difference?

Reply:
they're opposites: `\s` is whitespace`, `\S` is anything except whitespace. [See MDN](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Regular_Expressions/Character_Classes#types) 

But note that hasn't changed here, just the unnecessary square brackets